tsconfig.json

```diff
@@ -22,6 +22,6 @@
       "@/*": ["./*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "next.config.js"],
   "exclude": ["node_modules"]
```

Comment:
What is the purpose of placing JavaScript files in the configuration of TS?

Reply:
> What is the purpose of placing JavaScript files in the configuration of TS?

Automatically added by compiler. 
There's still a small issue with next.config.js, will have a look after.